applications/serial_lte_modem/overlay-carrier.conf

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_LWM2M_CARRIER=y
+
+# Workaround for known issue NCSDK-16916.
+# Set the size of the storage partition to a multiple of CONFIG_NRF_SPU_FLASH_REGION_SIZE.
+CONFIG_LWM2M_CARRIER_STORAGE_AREA_SIZE=0x8000
+
+# LwM2M carrier is only compiled for hard-float
+CONFIG_FP_HARDABI=y
+
+# SMS library
+CONFIG_SMS=y
+
+# PDN management support
+CONFIG_PDN=y
```

Comment:
SLM is AT-based so all PDN configurations are done by AT commands before powering on radio.
By including PDN, things won't work in networks like those in Japan, as extra PDN configuration items must also be provided (APN, username/password, etc). So far MCU configure them all by AT commands